### PR TITLE
Renovate update reference

### PR DESCRIPTION
**Pull request by TheMeier**
contains #765

### CODE REVIEW

File: renovate-config.json [hunk 1]
```diff
@@ -2,28 +2,51 @@
...
       "matchStrings": [
         "# renovate: depName=(?<depName>[^\\s]+?)\\s+String\\[1\\]\\s+\\$version\\s+=\\s+['\"]?(?<currentValue>[\\w+\\.]*)"
       ],
       "datasourceTemplate": "github-releases"
     }
-
+  ],
+  "postUpgradeTasks": {
+    "commands": [
+      "bundle install",
```
Comment (bastelfreak) [inline renovate-config.json:41]: there are github actions available to install the dependencies and setup ruby, can we use those as well?

File: .github/workflows/renovate.yml [hunk 2]
```diff
@@ -22,6 +16,8 @@ jobs:
         uses: renovatebot/github-action@v40.1.12
         with:
           configurationFile: renovate-config.json
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.PCCIBOT_PAT_REPO_PUBLIC_REPO }}
```
Comment (bastelfreak) [inline .github/workflows/renovate.yml:19]: Why is CI working in https://github.com/voxpupuli/puppet-prometheus/pull/776 when the new token isn't used yet? :thinking:

File: renovate-config.json (hunk 1 above)
Comment (TheMeier) [reply renovate-config.json:41]: it is inside the renovate image, so i guess not

File: .github/workflows/renovate.yml (hunk 2 above)
Comment (TheMeier) [reply .github/workflows/renovate.yml:19]: I ran the task manually on the branch which uses the token

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**